graphql/resolvers.js

```diff
@@ -194,6 +195,33 @@ const resolveFunctions = {
         return lobbyGroup && mapLobbyGroup(lobbyGroup, t)
       })
     },
+    branchs (_, {locale}, {loaders: {translations}}, info) {
+      const queriedFields = new Set(
```

Comment:
Remove since unused.

Reply:
Nope, branchs is used for the branch overview page